### PR TITLE
Adding sage and buckley to available hounds

### PR DESCRIPTION
**Pull request by Catelyn**
Adding sage and buckley to available hounds.

### CODE REVIEW

**Comment by zachwhaley** [comment]
Great job with the changes!

I'll check these tomorrow morning.

File: greyhounds/buckley.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: greyhound
+title: Buckley
+pic: buckley.png
+sex: MALE
+age: 4 YEARS
+color: black
+cats:
```
Comment (zachwhaley) [inline greyhounds/buckley.md]: Description says he is cat safe, which means cats should have a YES value

**Comment by zachwhaley** [comment]
Great job on the pics!! 

FYI: Thumbnails need to be exact squares for the greyhound grid to work, and they also don't need to be any bigger than 600x600 pixels

I've fixed the comments I made and tested that the site looks good.

This is ready to be merged :+1:

**Comment by Catelyn** [comment]
Thanks Zach! Full disclosure: Komson helped me figure this out :)

**Comment by zachwhaley** [comment]
Did y'all use the Github for Mac tool or command line Git?

**Comment by Catelyn** [comment]
Github for Mac

**Comment by zachwhaley** [comment]
Cool! and what did you use to edit the files?  I'm very curious :-)

Thank you,
Zach Whaley

**Comment by Catelyn** [comment]
We used TextEdit. No problem! How's the little one? The pics were adorable
you sent Tommi!

**Comment by zachwhaley** [comment]
Have a look at https://atom.io/ it does nifty things like highlight bits of
code and it can interact with Git directly.

Ellie is adorable ;-)  She's been getting better at sleeping at night, but
she still has trouble falling asleep without being held for ~2 hours.
So far, we've been enjoying her presence.  Oh and the dogs are being really
good around her; little sniffs here and there.

Thank you,
Zach Whaley

**Comment by Catelyn** [comment]
Nice- we'll check that out!

Aw she sounds sweet. I was surprised how dark her hair is! That's good about the dogs. Always a concern how they'll react, but sounds like they're being respectful. Glad to hear that. If we have a baby, I know my dog Reagan will not take it well haha.

**Pull request merged**